src/intrinsics.cpp

```diff
@@ -35,6 +35,18 @@ static void jl_init_intrinsic_functions_codegen(Module *m)
 #undef ALIAS
 }
 
+extern "C" {
+  JL_DLLEXPORT uint32_t jl_get_LLVM_VERSION(void);
```

Comment:
don't really need the braces for a single function

4 space indent in the function body